help/develop/javascript-development/datagrok-configuration.md

```diff
@@ -0,0 +1,39 @@
+# Datagrok configuration
```

Comment:
I think the title is missing: `<!-- Datagrok Configuration -->`. And strictly speaking, it's not the platform instance that is being configured, rather an environment for package development. We might want to rename this article.

Also, I was thinking about a new folder under `develop` called `tools`. What's your opinion on creating it and moving this file there? We've got a set of tools that we should cover: datagrok-tools, public packages for developers (DevTools and IconTool, which we probably integrate into the former), and inspector. And I have a draft of an article about common libraries that developers use when working on their packages. We can keep these things in one place.

Lastly, have we agreed on the following capitalization rules, or have they changed?

> In the document, use title-style capitalization for H1 and sentence-style capitalization for H2 and lower.